assignments/function-conversion.js

```diff
@@ -8,7 +8,11 @@
 // let anotherFunction = function (param) {
 //   return param;
 // };
-// anotherFunction("Example");
+// anotherFunction("Example"); 
+
+let anotherFunction = (param) => param 
+
+anotherFunction; 
```

Comment:
Remember, to invoke a function we use parentheses. If you wanted to invoke anotherFunction, you would need to do `anotherFunction()`